tests/estimators/speech_recognition/test_pytorch_deep_speech.py

```diff
@@ -121,6 +121,8 @@ def test_all(self, _test_all):
     @pytest.fixture(params=[False, True])
     def _test_all(self, request, setup_class):
         # Only import if deep speech module is available
+        import torch
```

Comment:
fixtures shouldn't be used in this way. This test code should simply be placed within the `def test_all(self, _test_all):` function directly. And instead use the `@pytest.mark.parametrize` marker to set a given param to various values see `test_loss_functions` in  `https://github.com/Trusted-AI/adversarial-robustness-toolbox/blob/dev_1.5.0/tests/estimators/classification/test_deeplearning_common.py`

Reply:
Solved.